uint64 timestamp			# time since system start (microseconds)

float32[3] magnetometer_ga		# Magnetic field in NED body frame, in Gauss
